pkgs/shells/fish/plugins/default.nix

```diff
@@ -15,7 +15,7 @@ lib.makeScope newScope (self: with self; {
 
   foreign-env = callPackage ./foreign-env { };
 
-  forgit-fish = callPackage ./forgit.nix { };
+  forgit = callPackage ./forgit.nix { };
```

Comment:
Nitpicks:
* it would be better to put the attribute rename in a separate commit,
* commit format should also be of the form: "fishPlugins.forgit: ...".